_core/smart/tutorial-counter.md

```diff
@@ -169,6 +170,8 @@ Let's get familiar with the tests to understand what the new smart contract shou
         (ok (var-get counter))))
     ```
 
+---
```

Comment:
I'd make bulletpoint 6 just a sentence, not part of the list. It suggests there's one more step after we congratulate the reader

Reply:
Done!